x/ibc/light-clients/solomachine/types/update.go

```diff
@@ -42,7 +42,12 @@ func checkHeader(clientState *ClientState, header *Header) error {
 	}
 
 	// assert currently registered public key signed over the new public key with correct sequence
-	data := HeaderSignBytes(header)
+	// TODO: get timestamp
```

Comment:
@colin-axner where do I get the timestamp from?

Reply:
you can get it from the consensus state, but I think the better thing to do is add a timestamp field to the header. If you do that I'm not sure you have sign over the timestamp separately (since it'd be contained in the header)